scripts/validate-mdx-links.ts

```diff
@@ -0,0 +1,93 @@
+#! /usr/bin/env node --experimental-strip-types --no-warnings=ExperimentalWarning
+
+/**
+ * @file This script validates the internal links in the MDX files.
+ * Note that it does not validate external links (e.g. to GitHub).
+ *
+ * Usage:
+ * ./scripts/validate-mdx-links.ts --cwd packages/web/docs
+ */
+import { globSync } from 'node:fs';
+import { stat } from 'node:fs/promises';
+import { dirname, resolve } from 'node:path';
+import { parseArgs } from 'node:util';
+import { printErrors, scanURLs, validateFiles, type DetectedError } from 'next-validate-link';
+
+const args = parseArgs({
+  options: {
+    cwd: { type: 'string', default: process.cwd() },
+    files: { type: 'string', default: 'src/content/**/*.mdx' },
```

Comment:
```suggestion
    files: { type: 'string', default: 'src/**/*.mdx' },
```

@dimaMachina I'm changing it to grab components and `pages` dir too.